src/supertux/statistics.cpp

```diff
@@ -333,8 +333,8 @@ Statistics::draw_ingame_stats(DrawingContext& context, bool on_pause_menu)
                                     : Statistics::perfect_color
                               );
     context.color().draw_surface_scaled(coin_icon,
-                                        Rectf(pos.x + width + 1.f, pos.y + 1.f,
-                                              pos.x + width + 16.f, pos.y + 16.f),
+                                        Rectf(pos.x + width + 3.f, pos.y - 5.f,
+                                              pos.x + width + 3.f + 32.f, pos.y - 5.f + 32.f),
```

Comment:
`Rectf(Vector(pos.x + width + 3.f, pos.y - 5.f), Sizef(32.f, 32.f))`.

Also to be done for other such instances in the same file.